src/BaseTemplates/StarterWeb/StarterWeb.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>$netframeworkversion$</TargetFramework>$if$ ($context$ == WebCoreFullFramework)
-    <RuntimeIdentifier>win7-x86</RuntimeIdentifier>$endif$
+    <RuntimeIdentifier>win7-x86</RuntimeIdentifier>$endif$$if$ ($netframeworkversion$ == netcoreapp1.1)
+    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>$endif$
```

Comment:
My understanding is that for MVC templates using netcoreapp1.1 the razor functionality will always be enabled.